tests/test_integrated_channels/test_api/test_sap_success_factors/test_views.py

```diff
@@ -70,10 +72,6 @@ def test_get(self, mock_current_request):
                          self.sap_config.sapsf_company_id)
         self.assertEqual(int(data.get('sapsf_user_id')),
                          self.sap_config.sapsf_user_id)
-        self.assertEqual(data.get('key'),
-                         self.sap_config.key)
-        self.assertEqual(data.get('secret'),
-                         self.sap_config.secret)
```

Comment:
can we assert this on the encrypted columns?

Reply:
@sameenfatima78 each time encryption is attempted is made in the getter a new value encrypted value is returned, which is not possible to calculate and use in assertion